packages/calcite-components/src/components/accordion-item/resources.ts

```diff
@@ -18,3 +18,7 @@ export const CSS = {
   iconEnd: "icon--end",
   headerContainer: "header-container",
 };
+
+export const IDS = {
```

Comment:
Sidebar: we can update our conventions on using ID if needed within shadow DOM since IDs are encapsulated.

Reply:
Whoops, forgot to comment earlier. 😅